bttn_peak.py

```diff
@@ -0,0 +1,172 @@
+from matplotlib.widgets import Button
+import matplotlib.pyplot as plt
+
+
+class Bttn_peak(Button):
```

Comment:
python class names should be [CapWord](https://www.python.org/dev/peps/pep-0008/#class-names).
And possible, avoid abreviations: `class PeakButton:`, also for file names. Could you fix when applicable?

Reply:
Addressed in: fba60de4e22a9c87712d817319b1bd1ad8e0a168 4d4e3058edcc30b5dd3bee3ec5938f126b624f47